go.mod

```diff
@@ -107,7 +107,7 @@ require (
 	github.com/prometheus/client_golang v1.20.5
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.62.0
-	github.com/quay/claircore v1.5.34
+	github.com/quay/claircore v1.5.35-0.20250207183226-79e00354c107
```

Comment:
do we plan on waiting for v1.5.35 to be cut?

Reply:
Thought process was to get this PR out there, reviewed, potentially merged, and update the claircore version prior to actual Stackrox release.

Reply:
https://github.com/stackrox/stackrox/pull/14135/commits/8335d6735cc4332b90356abd31840f8b6d171cbe updated to release cut today

Reply:
Generating an SBOM after the update shows the released `Claircore` version:

```json
{
  "spdxVersion": "SPDX-2.3",
  "dataLicense": "CC0-1.0",
  "SPDXID": "SPDXRef-DOCUMENT",
  "name": "sha256:088eea90c3d0a540ee5686e7d7471acbd4063b6e97eaf49b5e651665eb7f4dc7",
  "documentNamespace": "https://docker.io/library/nginx-f415f351-de3e-4d3b-9ba4-0469571c2fcc",
  "comment": "Tech Preview - generated for 'docker.io/library/nginx:latest'",
  "creationInfo": {
    "creators": [
      "Tool: Claircore-v1.5.35",
      "Tool: scanner-v4-matcher-4.7.x-731-g8335d6735c"
    ],
    "created": "2025-02-11T23:21:56Z"
  },
}
```